### ISSUE TITLE
Incorrect solution for a simple linear programming problem.

### ISSUE THREAD

**Issue by TsReaper** [coin-or/pulp#160]
The code below (using pulp 1.6.8) solves the linear programming problem:
min -x
s.t. -2x <= 0
This problem is evidently unbounded, but the pulp solver gives out a status of 1, indicating a bounded solution.

```python
from pulp import *

prob = LpProblem('test', LpMinimize)
x = LpVariable('x')

prob += -1 * x
prob += -2 * x <= 0

print(prob.solve())
```

**Comment by pchtsp** [comment]
It does look weird, indeed.

I generated the mps and lp files and they are both correctly generated:

For example the LP is the following:
```
\* test *\
Minimize
OBJ: - x
Subject To
_C1: - 2 x <= 0
Bounds
x free
End
```

It seems to be a CBC problem. If I try with CPLEX as solver, it gives the "correct" message:

```
import pulp

prob = pulp.LpProblem('test', sense=pulp.LpMinimize)
x = pulp.LpVariable('x')

prob += - x
prob += -2 * x <= 0

print(prob.solve(pulp.CPLEX_CMD(keepFiles=1)))  # -1
```

The log CPLEX generates is the following:

```
Log started (V12.8.0.0) Wed Nov 14 16:34:09 2018


Problem 'test-pulp.lp' read.
Read time = 0.00 sec. (0.00 ticks)
Dual infeasible due to empty column 'x'.
Presolve time = 0.00 sec. (0.00 ticks)
Presolve - Unbounded or infeasible.
Solution time =    0.00 sec.
Deterministic time = 0.00 ticks  (0.10 ticks/sec)

CPLEX Error  1217: No solution exists.
No file written.
```

So I guess you'd have to write in the CBC forums to ask what's going wrong in this particular case because at least pulp is doing the correct thing.

**Comment by jGaboardi** [comment]
Also gives the "correct" solution in `gurobi`:

```
Python 3.6.6 | packaged by conda-forge | (default, Jul 26 2018, 09:55:02) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.5.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: %paste
import pulp

prob = pulp.LpProblem('test', sense=pulp.LpMinimize)
x = pulp.LpVariable('x')

prob += - x
prob += -2 * x <= 0

print(prob.solve(pulp.GUROBI_CMD()))

## -- End pasted text --
Academic license - for non-commercial use only

Gurobi Optimizer version 8.1.0 build v8.1.0rc1 (mac64)
Copyright (c) 2018, Gurobi Optimization, LLC

Read LP format model from file /var/folders/tx/yn969dwd46l872m04ms5zqy80000gn/T/5904342aeaaf4382b2436808b6367077-pulp.lp
Reading time = 0.00 seconds
OBJ: 1 rows, 1 columns, 1 nonzeros
Optimize a model with 1 rows, 1 columns and 1 nonzeros
Coefficient statistics:
  Matrix range     [2e+00, 2e+00]
  Objective range  [1e+00, 1e+00]
  Bounds range     [0e+00, 0e+00]
  RHS range        [0e+00, 0e+00]
Presolve time: 0.00s

Solved in 0 iterations and 0.00 seconds
Infeasible or unbounded model

Unable to retrieve attribute 'X'

/Users/USER/miniconda3/envs/py3_at1866/lib/python3.6/site-packages/pulp/solvers.py:2015: UserWarning: GUROBI_CMD does provide good solution status of non optimal solutions
  warnings.warn('GUROBI_CMD does provide good solution status of non optimal solutions')
0
```

**Comment by pchtsp** [comment]
Good day!

I've tried giving the mps file generated when calling CBC to CPLEX and it returned 'unbounded' as a result; proving the file given to CBC was correctly generated.

This appears to be a problem to report to the [CBC mailing list](https://list.coin-or.org/mailman/listinfo/cbc) or the their [github page](https://github.com/coin-or/Cbc/issues). They will problably ask you for an example. You can generate an mps file by calling the solver with the keepFiles=True argument in pulp.
